### PR TITLE
[CI:DOCS] Update Master to reflect v2.0.5 release

### PR DESCRIPTION
**Pull request by mheon**
Adds release notes and updates current version in README.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,60 @@
 # Release Notes
 
+## 2.0.5
+### Features
+- Rootless Podman will now add an entry to `/etc/passwd` for the user who ran Podman if run with `--userns=keep-id`.
+- The `podman system connection` command has been reworked to support multiple connections, and reenabled for use!
+- Podman now has a new global flag, `--connection`, to specify a connection to a remote Podman API instance.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:7]: thought/suggestion:  It might be good to include a pointer to the online man page for more information.

**Comment by TomSweeneyRedHat** [comment]
LGTM
one possible change for consideration, I wouldn't hold for it.

**Comment by rhatdan** [comment]
LGTM

**Comment by mheon** [comment]
Rebased, CI should go green

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,60 @@
...
+
+### Changes
+- Podman's automatic systemd integration (activated by the `--systemd=true` flag, set by default) will now activate for containers using `/usr/local/sbin/init` as their command, instead of just `/usr/sbin/init` and `/sbin/init` (and any path ending in `systemd`).
+- Seccomp profiles specified by the `--security-opt seccomp=...` flag to `podman create` and `podman run` will now be honored even if the container was created using `--privileged`.
+
+### Bugfixes
+- Fixed a bug where the `podman play kube` would not honor the `hostIP` field for port forwarding ([#5964](https://github.com/containers/podman/issues/5964)).
+- Fixed a bug where the `podman generate systemd` command would panic on an invalid restart policy being specified ([#7271](https://github.com/containers/podman/issues/7271)).
+- Fixed a bug where the `podman images` command could take a very long time (several minutes) to complete when a large number of images were present.
+- Fixed a bug where the `podman logs` command with the `--tail` flag would not work properly when a large amount of output would be printed ((#7230)[https://github.com/containers/podman/issues/7230]).
```
Comment (zhangguanzhang) [inline RELEASE_NOTES.md:17]: `((#7230)[https://github.com/containers/podman/issues/7230])`
should be 
`([#7230](https://github.com/containers/podman/issues/7230))`

Comment (rhatdan) [reply RELEASE_NOTES.md:17]: @mheon Can you fix this so we can merge.

Comment (mheon) [reply RELEASE_NOTES.md:17]: Done

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**